webpack.config.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable */
+
```

Comment:
As highlighted by the previous reviewer, kindly fix linter errors instead of disabling linters. Linters are supposed to sort you out from having syntax and common errors and its bad practice to just disable them anytime you run into a problem.